Source/Fuse.Maps/iOS/MapViewDelegate.m

```diff
@@ -355,6 +360,30 @@ -(void)onTap:(UITapGestureRecognizer*)sender
 			CLLocationCoordinate2D coord = [_mapView convertPoint:l toCoordinateFromView:_mapView];
 
 			touchBlock(2, coord.latitude, coord.longitude);
+			[self checkOverlayTapped:coord];
+		}
+	}
+
+	-(void)checkOverlayTapped:(CLLocationCoordinate2D)coord
+	{
+		if (overlaySelectBlock == NULL)
+			return;
+
+		MKMapPoint mappoint = MKMapPointForCoordinate(coord);
+		for (id<MKOverlay> overlay in _mapView.overlays)
+		{
+			MKOverlayPathRenderer * renderer = (MKOverlayPathRenderer*)[_mapView rendererForOverlay:overlay];
+			CGPoint tapPoint = [renderer pointForMapPoint:mappoint];
+			if (CGPathContainsPoint(renderer.path, nil, tapPoint, false))
```

Comment:
Nit: I would prefer to add curly-braces around the `if`-body when nested statements contain curly-braces, though not a big deal.